Video_26/eater.cfg

```diff
@@ -0,0 +1,26 @@
+MEMORY {
+	ZP:		start = $0000, size = $0100, type = rw;
+  ROM: start = $8000, size = $7E00, type = ro, fill = yes, file = %O;
+  WOZMON: start = $FE00, size = $1FA, type = ro, fill = yes, file = %O;
+  RESETVEC: start = $FFFA, size = 6, type = ro, fill = yes, file = %O;
+	DUMMY:		start = $0000, size = $00FF, file = "";
+}
+
+SEGMENTS {
+	ZEROPAGE:	load = ZP, type = zp;
+	HEADER:		load = BASROM,	type = ro;
+	VECTORS:	load = BASROM,	type = ro;
+	KEYWORDS:	load = BASROM,	type = ro;
+	ERROR:		load = BASROM,	type = ro;
+	CODE:		load = BASROM,	type = ro;
+	CHRGET:		load = BASROM,	type = ro;
+	INIT:		load = BASROM,	type = ro;
+	EXTRA:		load = BASROM,	type = ro;
+	DUMMY:		load = DUMMY; # don't include
+
+  # customized
+  BIOS: load = "ROM", type = ro;
+  WOZMON: load = "WOZMON", type = ro;
+  RESETVEC: load = "RESETVEC", type = ro;
+}
+
```

Comment:
we can decide later whether tab size difference is good(because we can distinguish original code) or not(for a beauty).